manifests/s/sigoden/opscan/0.4.0/sigoden.opscan.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: sigoden.opscan
+PackageVersion: 0.4.0
+InstallerType: zip
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ReleaseDate: 2023-02-09
+Installers:
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: opscan.exe
+    PortableCommandAlias: opscan
+  InstallerUrl: https://github.com/sigoden/opscan/releases/download/v0.4.0/opscan-v0.4.0-i686-pc-windows-msvc.zip
+  InstallerSha256: DADF55DF83DC6A84B7B5F0C4F16B63DA40C847A5739BF8C3B95B4657D2AAEB64
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: opscan.exe
+    PortableCommandAlias: opscan
+  InstallerUrl: https://github.com/sigoden/opscan/releases/download/v0.4.0/opscan-v0.4.0-x86_64-pc-windows-msvc.zip
+  InstallerSha256: DA1C05B1E5CF5FA83875DA55A91964853888816DF6F35140B6A06B9E0F18E4A3
```

```suggestion
ReleaseDate: 2023-02-09
Installers:
- Architecture: x86
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: opscan.exe
    PortableCommandAlias: opscan
  InstallerUrl: https://github.com/sigoden/opscan/releases/download/v0.4.0/opscan-v0.4.0-i686-pc-windows-msvc.zip
  InstallerSha256: DADF55DF83DC6A84B7B5F0C4F16B63DA40C847A5739BF8C3B95B4657D2AAEB64
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
- Architecture: x64
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: opscan.exe
    PortableCommandAlias: opscan
  InstallerUrl: https://github.com/sigoden/opscan/releases/download/v0.4.0/opscan-v0.4.0-x86_64-pc-windows-msvc.zip
  InstallerSha256: DA1C05B1E5CF5FA83875DA55A91964853888816DF6F35140B6A06B9E0F18E4A3
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
This replaces the single dependency with architecture-specific dependencies.